bin/run-addon

```diff
@@ -94,7 +94,7 @@ done
 open=$binary # assumes Linux
 unamestr=$(uname)
 if [[ "$unamestr" = "Darwin" ]]; then
-  open="open $binary"
+  open=(open "${binary}")
```

Comment:
I don't quite understand why this needs to be an array.  Otherwise LGTM.

Reply:
I'm not much of a bash scripter. Suggestions welcome :-)

Reply:
The older code of having `open` as a string should work as far as I can tell.

Reply:
Oh, the spaces.  :satisfied: